packages/api/lib/writeRecords/write-granules.js

```diff
@@ -375,8 +375,13 @@ const _writeGranule = async ({
 }) => {
   let pgGranule;
 
-  log.info('About to write granule record %j to PostgreSQL', postgresGranuleRecord);
-  log.info('About to write granule record %j to DynamoDB', apiGranuleRecord);
+  log.info('About to write granule record %j to PostgreSQL', {
+    ...postgresGranuleRecord,
+    product_volume: postgresGranuleRecord.product_volume
+      ? String(postgresGranuleRecord.product_volume)
+      : undefined,
```

Comment:
yes, *although* at some point taken to it's logical conclusion we may be getting into more-or-less just modding serialization of BigInt into json....which is what we agreed we wanted to avoid on the API end of things.      
 
 That said, there's no reason we can't include a helper method in common/api/db/ for logging for this particular object key.